test/extensions/EXT/SPV_EXT_shader_atomic_float_/atomicrmw_fadd_half.ll

```diff
@@ -0,0 +1,34 @@
+; RUN: llvm-as < %s -o %t.bc
+; RUN: llvm-spirv --spirv-ext=+SPV_EXT_shader_atomic_float16_add %t.bc -o %t.spv
+; RUN: spirv-val %t.spv
+; RUN: llvm-spirv -to-text %t.spv -o - | FileCheck %s
```

Comment:
Why are we not checking reverse-generated LLVM code here?

Reply:
As @MrSidims commented earlier, reverse translation to check the result LLVM IR is already tested in test/extensions/EXT/SPV_EXT_shader_atomic_float_/AtomicFAddEXT.ll